website/content/en/docs/ansible/development-tips.md

```diff
@@ -6,107 +6,115 @@ weight: 12
 This document provides some useful information and tips for a developer
 creating an operator powered by Ansible.
 
-## Getting started with the k8s Ansible modules
+## Getting started with the Kubernetes Collection for Ansible
 
 Since we are interested in using Ansible for the lifecycle management of our
-application on Kubernetes, it is beneficial for a developer to get a good grasp
-of the [k8s Ansible module][k8s_ansible_module]. This Ansible module allows a
-developer to either leverage their existing Kubernetes resource files (written
-in YaML) or express the lifecycle management in native Ansible. One of the
-biggest benefits of using Ansible in conjunction with existing Kubernetes
-resource files is the ability to use Jinja templating so that you can customize
-deployments with the simplicity of a few variables in Ansible.
+application on Kubernetes, it is beneficial for a developer to get a good
+grasp of the [Kubernetes Collection for Ansible](https://galaxy.ansible.com/community/kubernetes).
+This Ansible collection allows a developer to either leverage their existing
+Kubernetes resource files (written in YAML) or express the lifecycle
+management in native Ansible. One of the biggest benefits of using Ansible in
+conjunction with existing Kubernetes resource files is the ability to use
+Jinja templating so that you can customize deployments with the simplicity of
+a few variables in Ansible.
 
-The easiest way to get started is to install the modules on your local machine
-and test them using a playbook.
+The easiest way to get started is to install the collection on your local
+machine and test them using a playbook.
 
-### Installing the k8s Ansible modules
+### Installing the Kubernetes Collection for Ansible
+
+To install the Kubernetes Collection, one must first install Ansible 2.9+.
+For example, on Fedora/Centos:
 
-To install the k8s Ansible modules, one must first install Ansible 2.9+. On
-Fedora/Centos:
 ```bash
 $ sudo dnf install ansible
 ```
 
 In addition to Ansible, a user must install the [OpenShift Restclient
-Python][openshift_restclient_python] package. This can be installed from pip:
+Python](https://github.com/openshift/openshift-restclient-python) package:
```

Comment:
Please, revert it. The links should be added in the bottom.

Reply:
Okay, I was thinking that maybe the target is only referenced once, we can just put it in place.
For people who work on this doc, scrolling to the bottom to find out the target is a bit inconvenient, unless the target is mentioned several times in the doc.

But anyways, I'll respect the team's convention on this.